src/Avalonia.Base/PropertyStore/ValueStore.cs

```diff
@@ -159,8 +158,9 @@ public IDisposable AddBinding<T>(DirectPropertyBase<T> property, IObservable<obj
         public void ClearLocalValue(AvaloniaProperty property)
```

Comment:
This method needs to be renamed now, it sounds like it only has an effect if there is a value with `BindingPriority.Local`.

Reply:
Changed to `ClearValue` to match the method on `AvaloniaObject`.